lib/index.js

```diff
@@ -22,7 +22,7 @@ $(document).ready(() => {
     if (event.target.nodeName == 'BUTTON') {
       var id = event.target.id.split('-')[2]
       foodsRequests.deleteFood(id)
-      foodsRequests.removeFoodRow()
+      removeFoodRow()
```

Comment:
Like the move towards organizing methods and grouping them where they should be.